ecs/templates/grafana_build.json.tpl

```diff
@@ -48,7 +48,11 @@
       },
       {
         "name": "GF_SERVER_PROTOCOL",
-        "value": "http"
+        "value": "{server_protocol}"
+      },
+      {
+        "name": "GF_SERVER_ROOT_URL",
+        "value": "${server_protocol}://${domain_name}"
```

Comment:
If this one is set to https, does it fix the links without breaking anything else?

Reply:
Pass. Will have to check locally. I've got a feeling it may cause issues having a different protocol defined in the the root url